dashboard/src/containers/LayoutContainer/LayoutContainer.ts

```diff
@@ -0,0 +1,17 @@
+import { connect } from "react-redux";
+import { RouteComponentProps } from "react-router";
+
+import Layout from "../../components/Layout";
+import { IStoreState } from "../../shared/types";
+
+interface IState extends IStoreState {
+  router: RouteComponentProps<{}>;
+}
+
+function mapStateToProps({ config: { featureFlags } }: IState) {
+  return {
+    UI: featureFlags.ui,
+  };
+}
+
+export default connect(mapStateToProps)(Layout);
```

Comment:
Note: you might to wait or my branch updating our redux/react-router dependencies etc. as you'll no longer need a container to access state (as it can be accessed via hooks). Similarly, updating connected-react-router forces adding the router state to the `IStoreState` as you've done manually here (ie. it'll already be done in my branch).

Reply:
I prefer to merge this before so I don't have conflicts since it's quite simple, in any case, the route thing is not needed.